frontend/talentsearch/src/js/components/reviewMyApplication/ReviewMyApplicationPage.tsx

```diff
@@ -0,0 +1,266 @@
+          title: intl.formatMessage({
+            defaultMessage: "Step 1",
+            description: "Breadcrumb for review application page.",
+          }),
+        },
+      ]}
+      navigation={{
+        currentStep: 1,
+        steps: [
+          {
+            path: directIntakePaths.reviewApplication(applicant.id),
+            label: intl.formatMessage({
+              defaultMessage: "Step 1: Review my profile",
+              description: "Navigation step in sign and submit page.",
+            }),
+          },
+          {
+            path: directIntakePaths.signAndSubmit(applicant.id),
+            label: intl.formatMessage({
+              defaultMessage: "Step 2: Sign and submit",
+              description: "Navigation step in sign and submit page.",
+            }),
+            disabled: !isApplicationComplete ?? false,
+          },
+        ],
+      }}
+      subtitle={jobTitle || intl.formatMessage(commonMessages.jobTitleNotFound)}
+    >
+      <UserProfile
+        applicant={applicant}
+        sections={{
+          myStatus: { isVisible: false },
+          hiringPools: { isVisible: false },
+          about: {
+            isVisible: true,
+            editUrl: `${directIntakePaths.aboutMe(
+              poolCandidateId,
+            )}?application=${poolCandidateId}`,
+          },
+          language: {
+            isVisible: true,
+            editUrl: `${directIntakePaths.languageInformation(
+              poolCandidateId,
+            )}?application=${poolCandidateId}`,
+          },
+          government: {
+            isVisible: true,
+            editUrl: `${directIntakePaths.governmentInformation(
+              poolCandidateId,
+            )}?application=${poolCandidateId}`,
+          },
+          workLocation: {
+            isVisible: true,
+            editUrl: `${directIntakePaths.workLocation(
+              poolCandidateId,
+            )}?application=${poolCandidateId}`,
+          },
+          workPreferences: {
+            isVisible: true,
+            editUrl: `${directIntakePaths.workPreferences(
+              poolCandidateId,
+            )}?application=${poolCandidateId}`,
+          },
+          employmentEquity: {
+            isVisible: true,
+            editUrl: `${directIntakePaths.diversityEquityInclusion(
+              poolCandidateId,
+            )}?application=${poolCandidateId}`,
+          },
+          roleSalary: {
+            isVisible: true,
+            editUrl: `${directIntakePaths.roleSalary(
+              poolCandidateId,
+            )}?application=${poolCandidateId}`,
+          },
+          skillsExperience: {
+            isVisible: true,
+            editUrl: `${directIntakePaths.skillsAndExperiences(
+              poolCandidateId,
+            )}?application=${poolCandidateId}`,
+            override: (
+              <>
+                {missingSkills && (
+                  <div data-h2-margin="base(x1, 0)">
+                    <MissingSkills
```

Comment:
There should be a grey box under this to match the other sections. 
![image](https://user-images.githubusercontent.com/8978655/187976729-bb331e23-3f54-497d-9a8f-b181e1ff65f0.png)

Reply:
Fixed by [Improve styling to match designs](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3796/commits/4c06217b89c30bf6548078c889c46d6de10ca560)